### DISCUSSION TITLE
After clicking text on a <Link> component in Gatsby the style of the text disappears.

### DISCUSSION THREAD

**Discussion by MarcelOlsen** [tailwindlabs/tailwindcss#2287] [Help]
I've created a header for my website. I'm using the <Link> tag since it's better than normal <a> but it's a Gatsby feature I believe. Anyway when I click a nav link the text inside it the style I've put on it disappears.
<img width="414" alt="" src="https://user-images.githubusercontent.com/45267579/92008127-088ada80-ed47-11ea-8d4b-dc0457724bc8.png">
<img width="398" alt="" src="https://user-images.githubusercontent.com/45267579/92008131-09bc0780-ed47-11ea-8dd6-727433fb4b64.png">

**Comment by lukebennett88** [comment]
I predominantly work in Gatsby, and the `Link` component definitely doesn't add any extra styles, so they must be leaking in from somewhere else.

2 things I can think of that might be causing this:
- Are you importing any custom CSS anywhere else in the project?
- The `Link` component takes an `activeClassName` prop — this could be overriding your Tailwind styles if you're using it.

**Reply by MarcelOlsen** [reply]
Thanks for the reply, I'm not using the `activeClassName` prop. I do use custom css but not for the <Link> tags, just for keeping the footer at the bottom of the page.
